frontend/src/app/inbox/ChatInbox.css

```diff
@@ -1,3 +1,14 @@
 .ChatInbox {
     margin-top: 5px;
 }
+.HiddenInbox {
+    background: var(--color-theme-contrast);
+    color: var(--color-text);
+    box-shadow: 0 0 5px 1px var(--color-theme-accent);
+    padding: 1.5em;
+    box-sizing: border-box;
+    display: flex;
+}
+.btn {
+    margin: 15px;
```

Comment:
Optional: Can you add a button hover rule for `cursor: pointer`? This is a standard way to indicate to users that something is clickable.